python/pyspark/sql/functions.py

```diff
@@ -882,6 +882,12 @@ def approx_count_distinct(col, rsd=None):
 def broadcast(df):
     """Marks a DataFrame as small enough for use in broadcast joins."""
```

Comment:
don't document PySpark boradcast. deprecate `ps.broadcast` alone. PySpark's `broadcast` has been there for a long time from since 1.6, and it doesn't much make sense to deprecate it for legacy users.